### PR TITLE
Created HTML structure for site pages

### PR DESCRIPTION
**Pull request by CBreul**
Added HTML skeletons for Index, The Party, NPCs and Monsters, and the Battle Log.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,9 +3,50 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>The Crew</title>
 </head>
 <body>
-    <iframe width="560" height="315" src="https://www.youtube.com/embed/zN9nsWOF430" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    <header>
+        <!--Begin title/header-->
+        <section>
+            <img src="images/TempLogo.jpg" style="float:left;" height="80" width="80" alt="An image of three red first-edition Dungeons and Dragons manuals on a white tablecloth, next to a blue bag of dice. The dice are spilling out of the bag, artfully arranged by the edge of the manuals.">
```
Comment (axamei) [inline index.html]: Let's wait for the CSS section to handle styling.

**Review by axamei — CHANGES_REQUESTED** [review]
Hi @CBreul it looks like you are missing an HTML file for locations.

File: index.html (hunk 1 above)
Comment (CBreul) [reply index.html]: Hi Professor Liauw @axamei ! Apologies - Locations was a page I thought I wanted but ended up not needing, and I forgot to remove it from the nav. I have removed references to Locations, as well as the styling from the header image.

**Review by axamei — APPROVED** [review]
👍

**Pull request merged**